sample-apps/radiology/lib/transforms/transforms.py

```diff
@@ -505,3 +506,35 @@ def __call__(self, data: Mapping[Hashable, NdarrayOrTensor]) -> Dict[Hashable, N
             if d.get(cache_key) is None:
                 d[cache_key] = copy.deepcopy(d[key])
         return d
+
+
+class OrientationGuidanceMultipleLabelDeepEditd(Transform):
+    def __init__(self, label_names=None):
+        """
+        Convert the guidance to the RAS orientation
+        """
+        self.label_names = label_names
+
+    def transform_points(self, point, affine):
+        """transform point to the coordinates of the transformed image
+        point: numpy array [bs, N, 3]
+        """
+        bs, N = point.shape[:2]
+        point = np.concatenate((point, np.ones((bs, N, 1))), axis=-1)
+        point = rearrange(point, "b n d -> d (b n)")
+        point = affine @ point
+        point = rearrange(point, "d (b n)-> b n d", b=bs)[:, :, :3]
+        return point
+
+    def __call__(self, data):
+        d: Dict = dict(data)
+        for key_label, val_label in self.label_names.items():
```

Comment:
`val_label` seems useless here, can use `_` or `for key in label_names.keys()` instead.

BTW, what should be the format of `val_label` in `self.label_names`?  Can you provide an example?

I don't know why label names need to be a dict.